src/analysis/wordcloud_leena.py

```diff
@@ -0,0 +1,120 @@
+import pandas as pd
+import matplotlib.pyplot as plt
+from wordcloud import WordCloud
+
+df = pd.read_csv("src/datasets/emscad_v1.csv", encoding="ISO-8859-1")
+
+category_mapping = {
+    "Business": ["business", "sales", "marketing", "accounting", "finance"],
+    "Human Resource": ["admin", "adminstrative"],
+    "Mass Communications": [
+        "advertising",
+        "entertainment",
+        "public relations",
+        "arts",
+        "creative design",
+    ],
+    "Hotel": ["tourism", "hotel"],
+    "Educator": ["teach", "training", "lecture"],
+    "Manufacturing": [
+        "process control",
+        "quality assurance",
+        "maintenance",
+        "manufacturing",
+        "purchasing",
+    ],
+    "Construction": [
+        "real estate",
+        "housing",
+        "constructions",
+        "architect",
+        "interior design",
+    ],
+    "Healthcare": ["pharmacy", "practitioner", "diagnosis"],
+    "Science": [
+        "agriculture",
+        "food technology",
+        "biology",
+        "chemistry",
+        "physics",
+        "biomedical",
+        "science",
+        "biotechnology",
+        "geology",
+    ],
+    "engineering": ["engineer", "mechanical", "electrical", "civil"],
+    "IT": ["IT", "information technology", "developer", "programmer"],
+}
+
+
+def categorise_job_title(title, category_mapping=category_mapping):
+    for category, keywords in category_mapping.items():
+        for keyword in keywords:
+            if str(keyword).lower() in str(title).lower():
+                return category
+    return "Other"
+
+
+# Apply the categorization function to the job titles and create a new 'Category' column
+df["Category"] = df["title"].apply(lambda x: categorise_job_title(x, category_mapping))
+
+df["IsOther"] = df["Category"] == "Other"
+category_counts = df["Category"].value_counts()
+
+df["Category"].value_counts()
+
+fraud_counts = df.groupby(["Category", "fraudulent"]).size().unstack().fillna(0)
```

Comment:
**Not bad Nice Graph! XD**
Maybe you can try to also sort them in order. Currently it is really jumbled up so its quite hard to see.
Also you can consider only displaying the top n number of categories since currently many categories' count is quite low, making it hard to read the graph and also it kinda doesn't contribute to the story that you want to tell your audience who are looking at the graph.